pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
I'm not getting what the root cause of this problem is. Can you provide the means to reproduce the problem? And perhaps describe the sequence of events.

We'll also need full stack trace of the error.